src/driftpy/types.py

```
@@ -315,6 +317,14 @@ class MarketType:
     Spot = constructor()
     Perp = constructor()
 
+def market_type_to_string(market_type: MarketType):
+    type_str = str(type(market_type))
+    if 'Perp' in type_str:
+        return 'perp'
+    elif 'Spot' in type_str:
+        return 'spot'
+    else:
+        return 'unknown'
```

id probably throw an error here